### ISSUE TITLE
Mac Apple Store

### ISSUE THREAD

**Issue by hspagnolo** [kengruven/strukt-bugs#50]
Hi Ken,

Strukt is not available anymore on the Swiss Mac Apple Store?

Thanks and regards
Henri

**Comment by kengruven** [comment]
Yes, it's true -- globally, not just in Switzerland.  Sorry I haven't been the best about communicating this.  (I just now realized I need to update the webpage.)  The Mac App Store simply became too much hassle for too little benefit.

If you encounter an issue that needs fixing, I'll figure out a way to get an update to you.

It's possible that Strukt as a Mac application will live on through some other distribution method.  Or the core technology will continue in some other form.  I haven't decided.  I'm working on other software which may supersede Strukt, but it's not my primary job, and my circumstances have changed a bit since 2012.
